NEWS

```diff
@@ -1,3 +1,82 @@
+Version 1.0.0 - xx December 2019
+
+- Add support for CUDA 10.2: QUDA 1.0.0 is supported on CUDA 7.5-10.2
+  using either GCC or clang compilers.  CUDA 10.x and either GCC >=
+  6.x or clang >= 6.x are highly recommended.
+
+- Signficant improvements to the CMake build system and removal of the
+  legacy configure build.
+
+- Added more targeted compilation options to constrain which
+  precisions and reconstruct types are compiled.  QUDA_PRECISION is a
+  cmake parameter that is a 4-bit number corresponding to which
+  precisions are enabled, with 1 = quarter, 2 = half, 4 = single and 8
+  = double, the default is 14 which enables double, single and half
+  precision.  QUDA_RECONSTRUCT is a 3-bit number corresponding to
+  which reconstruct types are enabled, with 1 = reconstruct-8/9, 2 =
+  reconstruct-12/13 and 4 = reconstruct-18, the default is 7 which
+  enables all reconstruct types.
+
+- Completely rewritten all dslash kernels using the accessor
+  framework.  This dramatically reduces code complexity and improve
+  performance.
+
+- New physics functionalithy added: gauge Laplace kernel, Gaussian
+  quark smearing, topological charge density.
+
+- QUDA can now be built to either utilize texture-memory reads or to
+  use direct memory accessing (cmake option QUDA_TEX).  The default is
```

Comment:
Isn't this backwards? Default is on for Pascal and earlier, off for Volta and Turing?

Reply:
Indeed, this is backwards.  And moreover, I didn't apply the change suggested in the notes that default is to have texture off for Volta/Turing, since getting it to work properly requires two calls to cmake (first sets arch, second sets tex).  So I just left the default as is for all arch.  Eventually texture will go away which make this a moot point.

Fixed.